src/app/i18n/locales/es.json

```diff
@@ -1399,7 +1407,8 @@
     "more": "Más",
     "continue": "Continuar",
     "keepCurrent": "Mantener actual",
-    "logOut": "Cerrar sesión"
+    "logOut": "Cerrar sesión",
+    "locked": "Bloqueado"
```

Comment:
the `locked` transtlation is gonna be used in another task 🈁

Reply:
Oops, I didn't put the question mark haha, I saw afterwards that it is for the next task